OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/immutability/ClassImmutability.scala

```diff
@@ -55,8 +60,9 @@ case object TransitivelyImmutableClass extends ClassImmutability {
     def meet(that: ClassImmutability): ClassImmutability = that
 }
 
-case class DependentlyImmutableClass(parameter: Set[String]) extends ClassImmutability {
+case class DependentlyImmutableClass(parameter: SortedSet[String]) extends ClassImmutability {
```

Comment:
There's so many lines where the parameter is initialized with SortedSet.empty, maybe it would be beneficial to put this as a default?